Makefile

```diff
@@ -68,6 +68,7 @@ checkfolders:
 .PHONY: noinstall install serialinstall mpiinstall cudainstall cudampiinstall aminstall
 
 install: $(INSTALLTYPES)
+	@cp -f $(toolsfolder)/sadguess $(exefolder)
```

Comment:
This should probably be in the serial install target since you use the serial quick executable for the guess generation.